README.md

```diff
@@ -12,3 +12,19 @@ This is a helper package that automatically installs and updates the
 
 Open configuration file using command palette with `Preferences: LSP-volar Settings` command or opening it from the Sublime menu (`Preferences > Package Settings > LSP > Servers > LSP-volar`).
 
+
+### Take over mode
+
+Allow LSP-volar to start in `*.ts` and `*.js` files.
+
+To enable take over mode, from the command palette select `Preferences: LSP-volar Settings` and put the following in `LSP-volar.sublime-settings`.
```

Comment:
It makes more sense to override per project.  And I want that behavor that you describe, but I could not achive it because of the following reason.

I cannot find a way to read the correct project data in `on_client_configuration_ready`

For example, if I have a st project file, with:
```
{
	"folders":
	[
		{
			"path": "."
		}
	],
	"settings": {
		"LSP": {
			"LSP-volar": {
				"settings": {
					"volar.takeOverMode.enabled": true
				}
			}
		}
	}
}
```

When I try to read the `volar.takeOverMode.enabled` here
https://github.com/sublimelsp/LSP-volar/pull/53/files#diff-08655555cc53ca79554ba180d1ae7e6ebe5c9b0a25e1772ce248fdd8b83dab34R26
```py
take_over_mode = configuration.get("settings", {}).get("volar.takeOverMode.enabled", "auto") # type: Union[str, bool]
```
I expect the `configuration.get("settings", {}).get("volar.takeOverMode.enabled", "auto")` to return `true` but it returns "auto" (the value defined in LSP-volar.sublime-settings).